src/convexhull.jl

```diff
@@ -0,0 +1,20 @@
+function upperhull(pset::PointSet) 
+
+  xs=Vector{Float64}[] 
+  xs=sort(coordinates.(pset), by=first)
+
+  convHull=[xs[1],xs[2]] 
+  
+  for i in 3:size(xs,1)
+    push!(convHull,xs[i])
+
+    # remove the second to last point in convHull while the last 3 points make a "left turn"
+    while size(convHull)[1]>2 && ∠(Point(xs[i-2]),Point(xs[i-1]),Point(xs[i]))<0 
```

Comment:
Can you please rewrite the comment with more details about the algorithm? Also, do you think it is possible to simplify the stop criterion? The first `size(convHul)[1] > 2` part seems redundant. Maybe it is always true?

Reply:
Actually, I don't think it is redundant. The collection might end up with just 2 points after  removal of the second to last point in the array. Aside from this I don't really know how to simplify the stopping criteria. The angle criterion seems necessary.